yogstation/code/modules/mob/living/carbon/human/species_types/preternis/organs.dm

```diff
@@ -0,0 +1,62 @@
+/obj/item/organ/eyes/preternis
+	name = "preternis eyes"
+	desc = "An experimental upgraded version of eyes that can see in the dark.They are designed to fit preternis"
+	see_in_dark = 8
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	actions_types = list(/datum/action/item_action/organ_action/use)
+	var/night_vision = TRUE
+
+/obj/item/organ/eyes/preternis/ui_action_click()
+	var/datum/species/preternis/S = owner.dna.species
+	if(S.charge < PRETERNIS_LEVEL_FED)
+		return
+	sight_flags = initial(sight_flags)
+	switch(lighting_alpha)
+		if (LIGHTING_PLANE_ALPHA_VISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+		if (LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+		if (LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE)
+			lighting_alpha = LIGHTING_PLANE_ALPHA_INVISIBLE
+		else
+			lighting_alpha = LIGHTING_PLANE_ALPHA_VISIBLE
+			sight_flags &= ~SEE_BLACKNESS
+	owner.update_sight()
+
+/obj/item/organ/eyes/preternis/on_life()
+	. = ..()
+	if(!ispreternis(owner))
+		qdel(src) //these eyes depend on being inside a preternis
```

Comment:
can't you like, just make them not work rather than making a specieschange mean they lose their eyes permanently?

Reply:
i tought so too but it gives them new eyes of the other species when they switch so this just affects transplants and its not meant to be in a preternis and this is wasted process cycles otherwise,tell me if its mandatory i change this